sdk/core/core-crypto/src/sha256.browser.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/// <reference lib="dom" />
+
+import { base64ToBytes, bufferToBase64 } from "./utils/base64.browser";
+import { bufferToHex } from "./utils/hex";
+import { utf8ToBytes } from "./utils/utf8.browser";
+
+let subtleCrypto: SubtleCrypto | undefined;
+
+/**
+ * Returns a cached reference to the Web API crypto.subtle object.
+ * @hidden
+ * @internal
+ */
+function getCrypto(): SubtleCrypto {
+  if (subtleCrypto) {
+    return subtleCrypto;
+  }
+
+  if (!self.crypto || !self.crypto.subtle) {
```

Comment:
good using `self` here, we should maybe add a webworker test

Reply:
Webworker tests added!